django_twitter_tsuji_v2/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for django_twitter_tsuji_v2 project.
+
+Generated by 'django-admin startproject' using Django 3.2.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from pathlib import Path
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+DEBUG = True
+ALLOWED_HOSTS = []
+AUTH_USER_MODEL = "accounts.User" 
+
+
+INSTALLED_APPS = [
+    'accounts',
+    'twitter',
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
+
+ROOT_URLCONF = 'django_twitter_tsuji_v2.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
```

Comment:
```suggestion
        'DIRS': [BASE_DIR / 'templates'],
```
として、各Appディレクトリ内ではなく、プロジェクト直下（accountsやtwitterディレクトリがあるところ）にtemplatesディレクトリを作成してください。そして、そのtemplatesディレクトリ内に各アプリ名と同じディレクトリを作成していください。